rae_gazebo/launch/gazebo.launch.py

```diff
@@ -8,15 +8,20 @@
 from launch.launch_description_sources import PythonLaunchDescriptionSource
 from launch.substitutions import LaunchConfiguration
 from launch.conditions import IfCondition
-from launch_ros.actions import Node
+from launch_ros.actions import Node, SetParameter
 
 def launch_setup(context, *args, **kwargs):
     rae_description_pkg = get_package_share_directory('rae_description')
     ros_gz_sim_pkg = get_package_share_directory('ros_gz_sim')
-    enable_localization = LaunchConfiguration(
-        'enable_localization', default='true')
+    namespace = LaunchConfiguration('namespace')
+    enable_localization = LaunchConfiguration('enable_localization')
     world_file = LaunchConfiguration('sdf_file').perform(context)
     print(f'world_file: {world_file}')
+
+    ns_srt = namespace.perform(context)
+    ns_srt = f'/{ns_srt}' if ns_srt else ''
```

Comment:
I'd either put that in a function or add a small comment explaining the behavior